core/adapters/bigquery.ts

```diff
@@ -16,7 +16,7 @@ export class BigQueryAdapter extends Adapter implements IAdapter {
   public publishTasks(
     table: dataform.ITable,
     runConfig: dataform.IRunConfig,
-    tableMetadata: dataform.ITableMetadata
+    tableMetadata?: dataform.ITableMetadata
```

Comment:
There's a slight issue with metadata; in order for the evaluation to be 100% accurate of evaluating what will run, metadata is required. This is annoying because acquiring metadata requires querying the entire warehouse (?), which obviously we don't want to happen every time a file is edited.